src/site/markdown/userguide.md.vm

```diff
@@ -148,6 +148,21 @@ import org.apache.commons.rdf.simple.SimpleRDFTermFactory;
 RDFTermFactory factory = new SimpleRDFTermFactory();
 ```
 
+If you don't want to depend up the concret implementation you can use a
+[ServiceLoader](http://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html)
+to lookup instances:
```

Comment:
concret -> concrete

"instances" -> "any implementations found on the classpath."